redash/models.py

```diff
@@ -673,13 +680,20 @@ def get_latest(cls, data_source, query, max_age=0):
 
     @classmethod
     def store_result(cls, org, data_source, query_hash, query, data, run_time, retrieved_at):
+        try:
+            data_scanned_information = json.loads(data)['data_scanned']
+        except:
```

Comment:
I'm not sure what this project's philosophy is but in general we try to avoid "naked except"s. Being explicit about the exceptions is preferred. In this case I'm guessing a `ValueError` or even a `KeyError`.